src/ui/public/vis/editors/default/default.js

```diff
@@ -2,11 +2,12 @@ import './sidebar';
 import './vis_options';
 import $ from 'jquery';
 
-
 import _ from 'lodash';
 import angular from 'angular';
 import defaultEditorTemplate from './default.html';
 
+import { VisEditorTypesRegistryProvider } from 'ui/registry/vis_editor_types';
+
```

Comment:
Either way, there shouldn't be any reason to import the default editor, ideally consumers will only import the editors register and depend on all editors so that they can be pluggable. If consumers import a specific editor they aren't pluggable.